packages/react/src/components/organisms/Header/_index.scss

```diff
@@ -0,0 +1,19 @@
+@use "~normalize-scss/sass/normalize";
```

Comment:
Can you separate out the global scss like normalize, elements, layout, fonts into a different scss file? And import it separately in the stories.js?

Reply:
This wouldn't be imported in the story itself, it'd be imported in the scss file.